triplea_maps.yaml

```diff
@@ -1677,7 +1674,7 @@
 - mapName: Zombieland
   mapCategory: EXPERIMENTAL
   url: https://github.com/triplea-maps/zombieland/archive/master.zip
-  version: 1
+  version: 2
```

Comment:
Zombieland contained two `version` keys with the values `1` and `2`, respectively.  Replaced with a single key for version 2.